src/index.css

```diff
@@ -61,16 +55,6 @@ img {
   opacity: 1 !important;
 }
 
-/* - Break everything out of their `position: absolute;` containers. */
-[data-testid="tweet"] * {
-  position: static !important;
-}
-
-/* - Hide all the placeholder aspect ratio elements. */
-[data-testid="tweet"] [style*="padding-bottom"] {
-  display: none !important;
-}
```

Comment:
🔥 Remove `position` and `display` overrides for image containers.

Now, let everything layout the way it does normally on twitter.com, without overrides.